src/dvr/dvr_inotify.c

```diff
@@ -70,7 +70,7 @@ pthread_t dvr_inotify_tid;
 
 void dvr_inotify_init ( void )
 {
-  atomic_set(&_inot_fd, inotify_init1(IN_CLOEXEC));
+  atomic_set(&_inot_fd, inotify_init());
```

Comment:
Yes, it is already contained in compat.h but not applied when I cross-compile PPC architectures and I don't know how I can add it for PPC architectures in addition. Applying it in general probably does not make sense. 

@jdierkse has solved the inotify issues via configuring with `--disable-inotify` argument in the past, but that circumvents a whole lot of code then...